ravenframework/Models/PostProcessors/Validations/PhysicsGuidedCoverageMapping.py

```diff
@@ -88,10 +90,23 @@ def run(self, inputIn):
     """
     dataDict = {self.getDataSetName(data): data for _, _, data in inputIn['Data']}
     names = list(dataDict.keys())#
-    evaluation ={k: np.atleast_1d(val) for k, val in self._evaluate(dataDict, **{'dataobjectNames': names}).items()}
+
+    if 'time' in inputIn['Data'][0][-1].coords.keys():
+      PCMVersion = 'Snapshot'
```

Comment:
FYI, the code standard for RAVEN is "camelBack", could you change "PCMVersion" to "pcmVersion"?

Reply:
The variable name has been corrected.